docs/src/index.md

```diff
@@ -1,8 +1,23 @@
 # TaylorModels.jl
 
-```@index
-```
+This package combines the `IntervalArithmetic.jl` and `TaylorSeries.jl` packages to provide **Taylor models**, i.e.
+Taylor polynomials with guaranteed error bounds to approximate functions.
 
-```@autodocs
-Modules = [TaylorModels]
-```
+An introduction is available in [this video from JuliaCon 2018](https://www.youtube.com/watch?v=o1h7BUW04NI).
+
+## Introduction to Taylor Models
+
+Taylor models provide a way to rigorously manipulate and evaluate functions using floating-point arithmetic. They have been widely used for validated computing: in global optimization and range bounding, for validated solutions of ODEs, rigorous quadrature, etc.
+
+A Taylor model (TM) of order $n$ for a function $f$ which is supposed to be $n + 1$ times continuously differentiable over an interval $[a,b]$, is a rigorous polynomial approximation of $f$. Specifically, it is a couple $(P, \Delta)$ formed by a polynomial $P$ of degree $n$, and an interval part $\Delta$, such that $f(x) − P(x) \in \Delta$, $\forall x ∈ [a,b]$. Roughly speaking, as their name suggests, the polynomial can be seen as a Taylor expansion of the function at a given point. The interval $\Delta$ (also called interval remainder) provides the validation of the approximation, meaning that it provides an enclosure of all the approximation errors encountered (truncation, roundings).
+
+### Authors
+- [Luis Benet](http://www.cicc.unam.mx/~benet/), Instituto de Ciencias Físicas, Universidad Nacional Autónoma de México (UNAM)
+- [David P. Sanders](http://sistemas.fciencias.unam.mx/~dsanders), Departamento de Física, Facultad de Ciencias, Universidad Nacional Autónoma de México (UNAM)
+
+## Bibliography
+- *Rigorous Polynomial Approximations and Applications*, Mioara Maria Joldes, Ecole normale supérieure de lyon - ENS LYON (2011)
```

Comment:
A minor suggestion: can you include the link to the actual thesis? It can be found at https://tel.archives-ouvertes.fr/tel-00657843